fusedrug/data/tokenizer/modulartokenizer/pretrained_tokenizers/configs/tokenizer_config.yaml

```diff
@@ -1,5 +1,5 @@
 paths:
-  tokenizers_path: "${oc.env:MY_GIT_REPOS}/fuse-drug/fusedrug/data/tokenizer/modulartokenizer/pretrained_tokenizers/" #tokenizer base work path
+  tokenizers_path: #tokenizer base work path
```

Comment:
I this legit yaml?  is this correct?

Reply:
I was expecting that someone will set it before running the script.
But maybe it's better to set it to null together with asserting that it's different from None at the beginning of the script. Setting it null will allow to override it in command line.